plugins/Marketplace/vue/src/PluginDetailsModal/PluginDetailsModal.vue

```diff
@@ -0,0 +1,508 @@
+                  <template v-if="pluginChangelogUrl">
+                    <template v-if="plugin.homepage">, </template>
+                    <a
+                      target="_blank"
+                      rel="noreferrer noopener"
+                      :href="externalRawLink(pluginChangelogUrl)"
+                    >{{ translate('CorePluginsAdmin_Changelog') }}</a>
+                  </template>
+
+                  <template v-if="plugin.repositoryUrl">
+                    <template v-if="plugin.homepage || pluginChangelogUrl">, </template>
+                    <a
+                      target="_blank"
+                      rel="noreferrer noopener"
+                      :href="externalRawLink(plugin.repositoryUrl)"
+                    >GitHub</a>
+                  </template>
+                </dd>
+              </div>
+
+              <div class="pair" v-if="pluginActivity && pluginActivity.numCommits">
+                <dt>{{ translate('CorePluginsAdmin_Activity') }}</dt>
+                <dd>
+                  {{ plugin.activity.numCommits }} commits
+
+                  <!-- eslint-disable-next-line max-len-->
+                  <template v-if="pluginActivity?.numContributors > 1">{{ ' ' + translate('Marketplace_ByXDevelopers', pluginActivity.numContributors) }}</template>
+                  <!-- eslint-disable-next-line max-len-->
+                  <template v-if="pluginActivity?.lastCommitDate">{{ ' ' + translate('Marketplace_LastCommitTime', pluginActivity.lastCommitDate) }}</template>
+                </dd>
+              </div>
+
+              <div class="pair" v-if="showLicenceName">
+                <dt>{{ translate('Marketplace_License') }}</dt>
+                <dd>
+                  <a v-if="pluginLatestVersion.license?.url" rel="noreferrer noopener"
+                     :href="pluginLatestVersion.license?.url"
+                     target="_blank">{{ pluginLatestVersion.license?.name }}</a>
+                  <span v-else>{{ pluginLatestVersion.license?.name }}</span>
+                </dd>
+              </div>
+
+              <template v-if="pluginSupport.length">
+                <div
+                  class="pair"
+                  v-for="(support, index) in pluginSupport"
+                  :key="`support-${index}`"
+                >
+                  <template v-if="support.name && support.value">
+                    <dt v-html="$sanitize(support.name)"></dt>
+                    <dd v-if="this.isValidHttpUrl(support.value)">
+                      <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        :href="externalRawLink($sanitize(support.value))"
+                      >{{ $sanitize(support.value) }}</a>
+                    </dd>
+                    <dd v-else-if="this.isValidEmail(support.value)">
+                      <a
+                        :href="`mailto:${$sanitize(support.value)}`"
```

Comment:
I went with what was originally in the template and these data points could technically be any text so I guess that's why it used `$sanitize` compared to just `encodeURIComponent` for the email where it was probably more likely only an email than anything else. I guess we can unify it around the email since the check for valid email should already be enough, and keep `$sanitize` for other supporting values.